Chapter2/chapter2.md

```diff
@@ -170,15 +170,15 @@ Of course, we can be more vague and tell NPM to install the latest version of `e
 
 The Figure 2-1 shows us results of running the aforementioned command. Please notice the path in Figure 2-1: `/usr/local/lib/node_modules/express-generator`. This is where, on Max OS X / Linux systems, NPM puts global modules by default. We verify the availability of Express.js CLI by running `$ express –V`.
 
-![alt](media/image1.png)

```

Comment:
for consistency with other chapters, it's lowercase. i fixed the case in the media folder for this chapter 2 in commit 730f3fb42c579c09c98f0bb11762527e272fa266